src/QualificationRunner.Core/Domain/GOFMergedPlot.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Core.Qualification;
+
+namespace QualificationRunner.Core.Domain
+{
+   public class GOFGroup
+   {
+      public ReferencingSimulation[] OutputMappings { get; set; }
+   }
+
+   public class GOFMergedPlot : IReferencingSimulations

```

Comment:
if we add a new plot type, we will need to add a new class here so that I can query the stuff properly. I tried with some querying on json but it was not very easy and also not easy to read